datafusion/spark/src/function/math/hex.rs

```diff
@@ -212,6 +215,16 @@ pub fn compute_hex(
 
                 Ok(ColumnarValue::Array(Arc::new(hexed)))
             }
+            DataType::Utf8View => {
+                let array = as_string_view_array(array)?;
+
+                let hexed: StringArray = array
```

Comment:
Here hexed is StringArray, why not using StringViewArray? Is it because we don't support it for the usage for this function?

Reply:
I saw that the return type of the function is String https://docs.databricks.com/gcp/en/sql/language-manual/functions/hex#returns